dobson/logusers.py

```diff
@@ -0,0 +1,16 @@
+import datetime
+
+import dobson.utils as utils
+
+
+def log_mac_addresses():
+    """Logs all mac addresses connected to the router into a file"""
+    mac_address = [mac.lower() for mac in utils.get_mac_addresses()]
```

Comment:
These should already be lowercase, given that `get_mac_addresses` converts to lowercase before replacing spaces with colons, so I don't think this line needs to be here.

Reply:
Okay, I'll fix it up.